ui/qt-app/ariadne-qt-app.cabal

```diff
@@ -0,0 +1,79 @@
+name:           ariadne-qt-app
+version:        0.2.0.0
+description:    Please see the README on Github at <https://github.com/serokell/ariadne#readme>
+homepage:       https://github.com/serokell/ariadne#readme
+bug-reports:    https://github.com/serokell/ariadne/issues
+license:        MPL-2.0
+license-file:   COPYING.md
+author:         @serokell
+maintainer:     Serokell <hi@serokell.io>
+build-type:     Simple
+cabal-version:  >= 1.10
+
+source-repository head
+  type: git
+  location: https://github.com/serokell/ariadne
+
+executable ariadne-qt
+  main-is: Main.hs
+  other-modules: Glue
+  ghc-options: -threaded -rtsopts -with-rtsopts=-N
+  build-depends:
+      ansi-wl-pprint
+    , ariadne-cardano
+    , ariadne-core
+    , ariadne-qt-lib
+    , async
+    , base >=4.7 && <5
+    , containers
+    , double-conversion
+    , ii-extras
+    , knit
+    , lens
+    , text
+    , universum
+    , vector
+    , vinyl
+  other-modules:
+      Paths_ariadne_qt_app
+  default-language: Haskell2010
+  default-extensions:
+      ApplicativeDo
+      NoImplicitPrelude
+      BangPatterns
+      ConstraintKinds
+      DataKinds
+      DeriveGeneric
+      DeriveFunctor
+      DeriveFoldable
+      DeriveTraversable
+      FlexibleContexts
+      FlexibleInstances
+      GADTs
+      GeneralizedNewtypeDeriving
+      LambdaCase
+      MonadFailDesugaring
+      MultiParamTypeClasses
+      MultiWayIf
+      NegativeLiterals
+      OverloadedStrings
+      PatternSynonyms
+      PolyKinds
+      RankNTypes
+      RecordWildCards
+      RecursiveDo
+      ScopedTypeVariables
+      StandaloneDeriving
+      TemplateHaskell
+      TupleSections
+      TypeApplications
+      TypeFamilies
+      TypeOperators
+      UndecidableInstances
+      ViewPatterns
+      EmptyCase
+      PartialTypeSignatures
+  ghc-options: -Wall -Wcompat -Werror
```

Comment:
I noticed that there is `ghc-options:` on line 20 and then here. Wouldn't it be better to merge them?